### PR TITLE
Pne

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-tls-dtls13.md [hunk 1]
```diff
@@ -487,6 +487,53 @@ current epoch implementations SHOULD use the most recent past epoch
...
+When the AEAD is based on AES, then the Mask is generated by
+computing AES-ECB on the first 16 bytes of the ciphertext:
+
+~~~~
+  Mask = AES-ECB(sn_key, Ciphertext[0..15])
+~~~~
+
+When the AEAD is based on ChaCha20, then the mask is generated
+by treating the first 12 bytes of the ciphertext as the
+Nonce and the next 4 bytes as the nonce:
```
Comment (chris-wood) [inline draft-ietf-tls-dtls13.md]: s/nonce/counter?

File: draft-ietf-tls-dtls13.md [hunk 2]
```diff
@@ -487,6 +487,53 @@ current epoch implementations SHOULD use the most recent past epoch
...
+~~~~
+
+[sender] denotes the sending side. The Secret value to be used is described 
+in Section 7.3 of {{I-D.ietf-tls-tls13}}.
+
+The encrypted sequence number is computed by XORing the leading
+bytes of the Mask with the sequence number. Decryption is
+accomplished by the same process.
+
+In some (rare) cases there the ciphertext may be less than 16 bytes.
```
Comment (chris-wood) [inline draft-ietf-tls-dtls13.md]: s/there the/the

File: draft-ietf-tls-dtls13.md [hunk 3]
```diff
@@ -487,6 +487,53 @@ current epoch implementations SHOULD use the most recent past epoch
...
+This cannot happen with most of the DTLS AEAD algorithms because
+the authentication tag itself is 16 bytes, however some algorithms
+such as TLS_AES_128_CCM_8_SHA256 have a shorter authentication tag,
+and in combination with a short plaintext, the result might be
+less than 16 bytes. In this case, implementations MUST pad the
+plaintext out in order to make a suitable-length ciphertext.
```
Comment (chris-wood) [inline draft-ietf-tls-dtls13.md]: Perhaps an example would be useful?

Comment (chris-wood) [reply draft-ietf-tls-dtls13.md]: And I assume one pads with zeros?

Comment (ekr) [reply draft-ietf-tls-dtls13.md]: this is just with the conventional padding mechanism.

**Pull request merged**